Makefile

```diff
@@ -57,6 +60,26 @@ shell-test-docker-tcp: DOCKER_HOST:=tcp://127.0.0.1:2375
 shell-test-docker-tcp:
 	./lstags nginx~/stable/
 
+shell-test-pullpush: start-local-registry push-to-local-registry stop-local-registry
+
+start-local-registry: REGISTRY_PORT:=5757
+start-local-registry:
+	docker rm -f registry-lstags &>/dev/null || true \
```

Comment:
`&>/dev/null` redirect is considered bashism.

Reply:
Yes, actually the whole line should be :fire: